src/main/java/Duke.java

```diff
@@ -1,10 +1,174 @@
+        allTasks[count] = new Todo(input);
+        echoTask();
+    }
+
+    private static void executeAddDeadline(String input) {
+        String[] split = splitDeadlineAndDate(input);
+        allTasks[count] = new Deadline(split[0], split[1]);
+        echoTask();
+    }
+
+    private static String[] splitDeadlineAndDate(String input) {
+        final int indexOfByPrefix = input.indexOf(PREFIX_BY);
+        String[] deadlineAndDate = new String[2];
+
+        deadlineAndDate[0] = input.substring(0, indexOfByPrefix).trim();
+        deadlineAndDate[1] = removePrefix(input.substring(indexOfByPrefix), PREFIX_BY).trim();
+        return deadlineAndDate;
+    }
+
+    private static void executeAddEvent(String input) {
+        String[] split = splitEventAndDates(input);
+        allTasks[count] = new Event(split[0], split[1], split[2]);
+        echoTask();
+    }
+
+    private static String[] splitEventAndDates(String input) {
+        final int indexOfFromPrefix = input.indexOf(PREFIX_FROM);
+        final int indexOfToPrefix = input.indexOf(PREFIX_TO);
+        String[] eventAndDates = new String[3];
+
+        eventAndDates[0] = input.substring(0, indexOfFromPrefix).trim();
+        eventAndDates[1] = removePrefix(input.substring(indexOfFromPrefix, indexOfToPrefix), PREFIX_FROM).trim();
+        eventAndDates[2] = removePrefix(input.substring(indexOfToPrefix), PREFIX_TO).trim();
+        return eventAndDates;
+    }
+
+    private static String removePrefix(String s, String prefix) {
+        return s.replace(prefix, "");
+    }
+
+    private static void echoTask() {
+        System.out.println(MESSAGE_ADD);
+        System.out.println(allTasks[count].toString());
+        count++;
+        String task = (count == 1) ? " task " : " tasks ";
+        System.out.println("Now you have " + count + task + "in the list.");
+    }
+
+    private static void executeList() {
+        if (count == 0) {
+            System.out.println(MESSAGE_EMPTY_LIST);
+            return;
+        }
+
+        System.out.println(MESSAGE_LIST_HEADER);
+        for (int i = 0; i < count; i++) {
+            System.out.println((i + 1) + ". " + allTasks[i].toString());
+        }
+    }
+
+    private static void executeMark(String taskNumber) {
+        int index = Integer.parseInt(taskNumber) - 1;
+        allTasks[index].markAsDone();
+        System.out.println(MESSAGE_MARK);
+        System.out.println(allTasks[index].toString());
+    }
+
+    private static void executeUnmark(String taskNumber) {
+        int index = Integer.parseInt(taskNumber) - 1;
+        allTasks[index].unmarkAsDone();
+        System.out.println(MESSAGE_UNMARK);
+        System.out.println(allTasks[index].toString());
+    }
+
+    private static void executeExit() {
+        System.out.println(MESSAGE_GOODBYE);
+        System.exit(0);
+    }
+
+    private static void executeError() {
+        System.out.println(MESSAGE_ERROR);
+    }
```

Comment:
Perhaps you could split these methods (and other methods that I may have missed out) into a separate file? Maybe a Parser.java and TaskManager.java?